ui/src/overview.tsx

```diff
@@ -190,6 +194,39 @@ function ParticipantsGroup(props: { group: tanagra.Group; index: number }) {
     ],
   });
 
+  const fetchGroupCount = useCallback(async () => {
+    const groupForFilter: tanagra.Group = {
+      id: props.group.id,
+      kind: tanagra.GroupKindEnum.Included,
+      criteria: props.group.criteria,
+    };
+
+    const cohortForFilter: tanagra.Cohort = {
+      id: cohort.id,
+      name: cohort.name,
+      underlayName: cohort.underlayName,
+      groups: [groupForFilter],
```

Comment:
Inline `groupForFilter`. It short enough there's no need for a separate variable.

Reply:
Done.